api/src/main/java/jakarta/config/Configuration.java

```diff
@@ -25,13 +25,47 @@
 import java.lang.annotation.Target;
 
 /**
- * <p>
- *     The annotation to be used to mark the interface to be handled by the injection support.
- * </p>
+ * <p>Indicates that the annotated class is a <em>configuration interface class</em> to be handled by the Jakarta Config
+ * implementation injection support.</p>
+ *
+ * <p>The <em>configuration interface class</em> is resolved with portion of application's
+ * <em>persistent configuration</em> identified by <em>configuration path</em>.
+ *
+ * <p>The terms <em>configuration interface class</em>, <em>configuration key</em>, <em>configuration path</em>, <em>canonical
+ * representation</em>, <em>persistent configuration</em>, <em>resolve</em>, and others are used here as defined in
+ * the Jakarta Config specification.</p>
+
+ * <p><strong>\u26A0 Caution:</strong> you are reading an incomplete draft specification that is subject to change.</p>
```

Comment:
`configuration interface class` does not make sense to me. I suggest to update to `configuration interface`.

Reply:
Indicates that the annotated class is a <em>configuration interface class</em> to be handled by the Jakarta Config implementation injection support. 
Does not read well. I suggest to update to: 
This annotation is to mark an interface to contain configurations. The configurations will be injected by the implmentation of Jakarta Config.